events/README.md

```diff
@@ -0,0 +1,472 @@
+  - None
+- **Body**
+  - Optional: `name=[string]` (`name` is the name of the event which is case-insensitive. Accepted characters are `a-z, A-Z, 0-9, and . - : _`)
+  - Optional: `description=[string]` (`description` describes the usage of the event.)
+  - Optional: template_id=[string] (template_id which can be found from dashboard)
+  - Optional: region=[string] (region in which you want to create event.)
+  ```json
+  {
+    "name": "new-event-1662723668",
+    "description": "This is a sample description for the event",
+    "template_id": "<template_id that you wish to associate w/ the event>",
+    "region": "in"
+  }
+  ```
+- **Headers**
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```json
+      {
+        "id": "<id>",
+        "name": "<event_name>",
+        "enabled": true,
+        "description": "This is a sample description for the event",
+        "customer": "<customer_id>",
+        "recording_info": {
+          "enabled": false
+        },
+        "template_id": "<template_id>",
+        "region": "in",
+        "created_at": "YYYY-MM-DDTHH:MM:SS.sssZ",
+        "updated_at": "YYYY-MM-DDTHH:MM:SS.sssZ"
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content**
+      `{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't create event. Please try again later'`
+
+## GET /events
+
+Get all the created events.
+
+- Params:
+  - enabled: true/false
+  - hits: Number of events in one response
+  - offset: Start of response
+    - ex: For 1-10 hits: 10, offset: 0; 11-20 hits: 10, offset: 10;
+- Query
+  - None
+- Body
+  - None
+- Headers
+  - Content-Type: application/json
+  - Authorization: Bearer <management_token>
+- **Cookie**
+  - rds-session: `<JWT>`
+- **Success Response:**
+  - **Code:** 200
+    - **Content:**
+      ```jsx
+      {
+      	"limit": 10,
+      	"data": [
+      		<event_object>,
+      		<event_object>,
+      	]
+      }
+      ```
+- **Error Response:**
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 500
+    - **Content**
+      `{ 'statusCode': 500, 'error': 'Internal server error', 'message': 'Couldn't get events. Please try again later'`
+
+## POST - /events/join
+
+Generate an auth token for a peer to join a event.
+
+To join the event (for guests, host, maven, moderator)
+
+- **Params:**
+  - None
+- **Query**
+  - None
+- **Body**
+  - Required: **`event_id=[string]`** (The ID of the event to which the user belongs)
+  - Required: **`user_id=[string]`** (The ID of the user for whom the token is being generated)
+  - Required: **`role=[string]`** (The role of the user in the event)
```

Comment:
can the user send only 1 role or multiple roles at a time ?

Reply:
1 role